docs/section19/19-13.md

```diff
@@ -0,0 +1,75 @@
+        - Reset all_design_setpoints_delta_Ts_are_per_reqs boolean to true: `all_design_setpoints_delta_Ts_are_per_reqs = true`  
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `if any(baseline_system_type_compare(sys_type, target_sys_type, false) for target_system_type in target_system_types):`      
+            - Get list of zones served by the hvac system: `zones_list_hvac_sys_b = dict_of_zones_and_terminal_units_served_by_hvac_sys_b[hvac.id]["ZONE_LIST"]`  
+            - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_b = dict_of_zones_and_terminal_units_served_by_hvac_sys_b[hvac.id]["ZONE_LIST"]["Terminal_Unit_List"]`  
+
+            Below determines the zone setpoint to use in the delta T calculation for systems with multiple zone thermostat setpoints, use the design set point that will result in the lowest supply air cooling set point or highest supply air heating set point: 
+            - Reset the design_thermostat_cooling_setpoint variable from the previous HVAC system: `design_thermostat_cooling_setpoint = +infinity`  
+            - Reset the design_thermostat_heating_setpoint variable from the previous HVAC system: `design_thermostat_heating_setpoint = -infinity`  
+            - Cycle through each zone associated with the HVAC system to get the lowest design_thermostat_cooling_setpoint and the highest design_thermostat_heating_setpoint: `for zone in zones_list_hvac_sys_b:`  
+                - Check if the zone_design_thermostat_cooling_setpoint is less than the design_thermostat_cooling_setpoint: `if zone.zone_design_thermostat_cooling_setpoint < design_thermostat_cooling_setpoint:` 
+                    - Set the design setpoint equal to the zone design setpoint: `design_thermostat_cooling_setpoint = zone.zone_design_thermostat_cooling`  
+                - Check if the zone_design_thermostat_heating_setpoint is greater than the design_thermostat_heating_setpoint: `if zone.zone_design_thermostat_heating_setpoint > design_thermostat_heating_setpoint:`  
+                    - Set the design setpoint equal to the zone design setpoint: `design_thermostat_heating_setpoint = zone.zone_design_thermostat_heating`  
+            
+            Below checks for a delta T of 20F:
+            - For each zone in the list of zones served by the hvac system: `for zone in zones_list_hvac_sys_b:`  
+                - Reset zone has lab space boolean variable: `zone_has_lab_space = false`  
+                - For each space in zone: `for space in zone.spaces:`
+                    - Check if space is a lab based on lighting_space_type: `if space.lighting_space_type in ["LABORATORY_EXCEPT_IN_OR_AS_A_CLASSROOM"]: zone_has_lab_space = true`
```

Comment:
So are we assuming that if there is at least one space that is a lab, the system design flow rate must be based on 17F instead of 20? 
We are excluding "Preschool through 12th grade, laboratory, and shop classrooms" space type, but I think it's OK.

Reply:
That is the what I assumed here. Its not clear from the language what the correct answer is.

Also, if no space types are defined it defaults to no lab. Maybe I should add a caveat if no all space types are defined, "Pass unless the hvac system serves a lab"?

Reply:
Sounds good on counting a zone with at least one lab space as a lab zone - this is consistent with the analogous computer room interpretation. Also OK with excluding lab classrooms. 

I am also OK with assuming that space is not a lab if space type is missing. Alternatively, we could make the opposite assumption for office building types (and perhaps also health-care clinic or hospital), but that will result in "undeterimined" flag for a lot of projects which is undesirable. Plus modeling higher delta T would result in more efficient baseline. I like your idea of adding note to Pass outcome.

Reply:
All set, no changes needed.